opensrp-app/src/main/java/org/smartregister/SyncConfiguration.java

```diff
@@ -152,11 +152,10 @@ public boolean hasExtraSettingsSync() {
 
     /**
      * Allows the user to define extra settings sync parameters
-     *
-     * @return settingsParams {@link List<String>}
+     * @return
      */
-    public List<String> getExtraSettingsParameters() {
-        return new ArrayList<>();
+    public String getExtraSettingsParameters() {
```

Comment:
Do we maybe want to overload this public method for backward compatibility and mark the former one with an annotation for `@Deprecated`?

Reply:
I will create another project. we can not overload coz of the different retutn types